apim-operator/deploy/sampe-definitions/security_definitions.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: wso2.com/v1alpha1
```

Comment:
We need to add the license header. Also, folder name is wrong. It is called sampe-definitions.

Reply:
done